src/test/run-make-fulldeps/pgo-branch-weights/Makefile

```diff
@@ -15,21 +15,29 @@ ifdef IS_MSVC
 COMMON_FLAGS=-Cpanic=abort
 endif
 
+# For some very small programs GNU ld seems to not properly handle
+# instrumentation sections correctly. Neither Gold nor LLD have that problem.
+ifeq ($(UNAME),Linux)
```

Comment:
Windows-gnu CI jobs will still use GNU LD version 2.27, is it recent enough to not have this bug?

Reply:
I think this is an ELF specific issue.